### ISSUE TITLE
Close share popup programmatically

### ISSUE THREAD

**Issue by WillStone** [EddyVerbruggen/SocialSharing-PhoneGap-Plugin#245]
Hi, I need to close the share popup programmatically when my app logs out/times out. How can I do it in both iOS and Android? Thanks

**Comment by NickAdriaenssens** [comment]
I would also like to see this functionality.

**Comment by akadebnath** [comment]
Yes. I also need this. The sharing function works perfectly. However, I want that is the user has not chosen a sharing method in 5 seconds, the popup will close automatically. Not a very common scenario, but it will be very helpful. Can anyone please add this?
